btm/src/main/java/bitronix/tm/resource/jms/DualSessionWrapper.java

```diff
@@ -324,6 +324,19 @@ public MessageConsumer createConsumer(Destination destination, String messageSel
         return messageConsumer;
     }
 
+    @Override
+    public MessageConsumer createSharedConsumer(Topic topic, String s) throws JMSException
```

Comment:
make param names consistent with the rest of the methods (is `s` here `selector`?)